Tests/Unit test preprocessing logic.py

```
@@ -0,0 +1,51 @@
+
+# COMMAND ----------
+
+test_case = TestPreProcess()
+test_case.setUpClass()
+
+# COMMAND ----------
+
+try:
+    test_case.test_append_headers()
+    print("test_append_headers passed!")
+except AssertionError as e:
```

Ruff

Local variable `e` is assigned to but never used

Remove assignment to unused variable `e`

(F841)